src/entry.cpp

```diff
@@ -637,6 +637,7 @@ namespace {
 	std::string entry::to_string() const
 	{
 		std::string ret;
+		if(type() == dictionary_t) ret.reserve(280);
```

Comment:
As I pointed out in the other comment, the rationale for "280" seems suspect. But it starting at a larger size will still probably significantly cut down on the number of times it will have to grow.
would you mind adding a space after the "if" though?